### ISSUE TITLE
Credentials not working with Azure DevOps & personal token

### ISSUE THREAD

**Issue by Kizmar** [desktop/desktop#10078]
### Describe the bug
GitHub Desktop just updated to version 2.5.3. I tried to push some changes to an ADO git repo with the same credentials I was using earlier and it's not accepting the credentials. Desktop keeps popping up the login dialog. I'm using a personal token for the password, have been. This was working prior to the update.

I was able to push the changes using vscode with the same credentials, so I know the credentials are valid.

As a side note: Desktop has been forcing me to enter repo credentials every day instead of storing them. It started doing that a while ago - used to store them until they were no longer valid. Once I log in it saves them, but then when I start working the next day, I have to enter them again. Is that by design?

### Version & OS
GitHub Desktop version 2.5.3
Windows 10 2004

### Steps to reproduce the behavior
Attempt to push changes to an Azure DevOps git repository using a personal token as the password.

### Expected behavior
Doesn't endlessly pop up the login dialog.

### Actual behavior
Endlessly pops up the login dialog.

**Comment by steveward** [comment]
Thanks for the detailed report @Kizmar. I'm going to take a look and see if I can reproduce this issue as well.

**Comment by steveward** [comment]
@Kizmar I was able to clone and push to an Azure DevOps repository without issue. Have you tried generating a new token to see if that gets things working?

It may also be helpful to check in the Credential Manager -> Windows Credentials to see what is being stored under `dev.azure.com`.

**Comment by Kizmar** [comment]
I originally cloned this using the "<company>.visualstudio.com" domain as the repository remote URL. Changed that to the "dev.azure.com" remote URL and still had the issue. I did end up creating a new token while on ADO using the "dev.azure.com" domain and GitHub Desktop was able to authenticate.

I hesitated creating a new token because authentication works in both Visual Studio and vscode using the old token.

At any rate... it seems to be working again. Thanks. :)
